### PR TITLE
Adds 'pronouns' field to /admin_members

### LINKED ISSUE #138: Add "pronouns" field to admin page
On the page with all the admins https://www.rubyonrails.link/admin_members
We would like to add a "pronouns" field.
The way I see this working is by adding a field "pronouns" (string) to [SlackUser](https://github.com/railslink/railslink/blob/master/app/models/slack_user.rb) and displaying it on the page
`pronouns` should be nullable, and when it is null it should not be displayed.
On the page, adding a `p` tag with the preferred_pronouns field provides a desirable style:
image
But you're welcome to style it how you want! All pronouns must be styled to same (nothing like "blue for boys, pink for girls" as this is exclusionary to non-binary and non-conforming individuals)
I looked on schema.org and wasn't able to find any practical schema to be used. Gender exists but not "pronouns" (https://schema.org/gender)

### PR DESCRIPTION
**Pull request by srijan-kapoor**
Closes #138 

Screenshot 2021-03-25 at 1 59 20 PM

### CODE REVIEW

**Comment by srijan-kapoor** [comment]
@grepsedawk can you please review this one? Thanks

The CI build is failing due to the versions of mimemagic gem prior to `0.3.6` being yanked. As a workaround, maybe we can try this https://github.com/rails/rails/issues/41757#issuecomment-808024283. Thoughts?

I can send another PR with this change.

File: app/views/pages/admin_members.html.slim [hunk 1]
```diff
@@ -15,4 +15,4 @@ main.admin-members
                 h3.title
                   = member.name
                 = link_to "@#{member.username}", "slack://user?team=#{ENV["SLACK_TEAM"]}&id=#{member.uid}", class: "has-text-primary"
-
+                p = member.pronouns
```
Comment (meagar) [inline app/views/pages/admin_members.html.slim:18]: Is it desirable to output an empty `<p>` tag here, in the case where the attribute is empty?

If not, something like this:

```
- if member.pronouns.present?
  p = member.pronouns
```

Comment (grepsedawk) [reply app/views/pages/admin_members.html.slim:18]: Hmm, perhaps yes: for spacing?

**Review by grepsedawk — APPROVED** [review]

**Comment by grepsedawk** [comment]
Looks good, but let's wait to deploy until there's a "real" fix for this ~~ since this should be fixed soon

**Comment by grepsedawk** [comment]
@srijan-kapoor Really nice work on this! Thank you.

**Comment by srijan-kapoor** [comment]
Thanks @grepsedawk for merging this 🙌🏽

**Pull request merged**